build.csx

```diff
@@ -246,7 +249,7 @@ public void RunDotNetTests(IEnumerable<string> testDirectories)
     foreach (var testDirectory in testDirectories)
     {
         var xml = Path.GetFullPath(Path.Combine(testsDirectory, Path.GetFileName(testDirectory) + ".TestResults.xml"));
-        Cmd(testDirectory, "dotnet", $"test -c Release -nologo -notrait \"explicit=yes\" -xml {xml}");
+        Cmd(testDirectory, "dotnet", $"xunit -configuration Release -nologo -notrait \"explicit=yes\" -xml {xml}");

```

Comment:
Is xunit adding the framework to the output path? I see `artifacts\tests\FakeItEasy.IntegrationTests.TestResults-net461.xml, but cannot tell how it was generated from this.